src/components/Toolbar.js

```diff
@@ -1,16 +1,23 @@
 import React from "react";
+import Radium from "radium";
+import StopButton from "stopButton" //??? naming convention?
```

Comment:
The file should have the same name as the default export. Also, you should actually be running the app to make sure everything works. Webpack should be giving you an error for this line.  It should be:
```javascript
import StopButton from './StopButton';
```